packages/pglite/src/pglite.ts

```diff
@@ -574,7 +574,10 @@ export class PGlite
    * @param message The postgres wire protocol message to execute
    * @returns The direct message data response produced by Postgres
    */
-  execProtocolRawSync(message: Uint8Array) {
+  execProtocolRawSync(
+    message: Uint8Array,
+    dataTransferContainerOverride?: DataTransferContainer,
+  ) {
```

Comment:
```suggestion
  execProtocolRawSync(
    message: Uint8Array,
    { dataTransferContainer?: DataTransferContainer } = {},
  ) {
```

I would make this an options object like we have on the others.